husky_components/tabs/main.js

```diff
@@ -54,7 +55,8 @@ define(function() {
             preSelectEvent: {
                 enabled: false,
                 triggerSelectItem: true
-            }
+            },
+            tabChangeAllowed: null
```

Comment:
Another solution would be to only pass a function which returns true or false, and check its return value in here. Then we wouldn't have to use deferreds here, which makes it easier to use.